### PR TITLE
feat(management/application): add Token Lifetime option to the action list

### PR DESCRIPTION
**Pull request by yusualhashash**
## How has this been tested?

- [ ] Prod
- [x] Development

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] Added all translations, no hard coded text. 
- [ ] Updated the UI (if needed).
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by ertugrulcan-ays — APPROVED** [review]

File: apps/web/src/app/[lang]/app/actions/api-requests.ts [hunk 1]
```diff
@@ -101,6 +103,7 @@ export type GetDetailTableDataTypes = Exclude<
   | "tags"
   | "editions"
   | "billing"
+  | "applications"
```
Comment (a0m0rajab) [inline apps/web/src/app/[lang]/app/actions/api-requests.ts:106]: It's out of scope of this PR, but we need to unify this, it's a bit overwhelming and hard to follow this way.

**Review by a0m0rajab — APPROVED** [review]
LGTM 👍
